### PR TITLE
Line-of-sight effects tutorial notebook

### PR DESCRIPTION
**Pull request by nataliehogg**
Addition of a notebook showing how the SinglePlaneLOS class can be used to add line-of-sight effects when generating and fitting images.

### CODE REVIEW

File: Notebooks/LineOfSightEffects/line-of-sight_effects_tutorial.ipynb [hunk 1]
```diff
@@ -0,0 +1,1542 @@
+{
```
Comment (sibirrer) [inline Notebooks/LineOfSightEffects/line-of-sight_effects_tutorial.ipynb:1]: <p>can you put the links to the papers in markdown hyperlinks?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lenstronomy/lenstronomy-tutorials/pull/14/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Comment (sibirrer) [inline Notebooks/LineOfSightEffects/line-of-sight_effects_tutorial.ipynb:1]: <p>'LOS' , missing '</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lenstronomy/lenstronomy-tutorials/pull/14/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Comment (sibirrer) [inline Notebooks/LineOfSightEffects/line-of-sight_effects_tutorial.ipynb:1]: <p>do you need this block in the tutorial (even before actually demonstrating the features)? The error message returned look quite good themselves, even without the tutorial. Also, if you try to run all of the notebook, it gets interrupted by this error message</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lenstronomy/lenstronomy-tutorials/pull/14/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Comment (sibirrer) [inline Notebooks/LineOfSightEffects/line-of-sight_effects_tutorial.ipynb:1]: <p>same comment as for the block above, the error message looks quite self-explanatory and it breaks the flow of the notebook</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lenstronomy/lenstronomy-tutorials/pull/14/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Comment (sibirrer) [inline Notebooks/LineOfSightEffects/line-of-sight_effects_tutorial.ipynb:1]: <p>latex does not show all the formulae in this NBreview page.</p><p>Also, do you want to give an explicit reference to the literature for this model?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lenstronomy/lenstronomy-tutorials/pull/14/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>

Comment (sibirrer) [inline Notebooks/LineOfSightEffects/line-of-sight_effects_tutorial.ipynb:1]: <p>perhaps you can cross-link to the SimulationAPI notebook? Just so the viewer/user can orientate that some of these features being used and explained have also other dedicated notebooks</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lenstronomy/lenstronomy-tutorials/pull/14/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>

Comment (sibirrer) [inline Notebooks/LineOfSightEffects/line-of-sight_effects_tutorial.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #19.    print('The measured relative delays are {}.'.format(dt_measured))
</pre><p>do you want to add the units days?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lenstronomy/lenstronomy-tutorials/pull/14/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='46'/>

Comment (sibirrer) [inline Notebooks/LineOfSightEffects/line-of-sight_effects_tutorial.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #25.    print('The time delay distance is {}.'.format(ddt))
</pre><p>do you want to add the units Mpc?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lenstronomy/lenstronomy-tutorials/pull/14/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='46'/>

**Review by sibirrer — CHANGES_REQUESTED** [review]
Thank you very much @nataliehogg !
This looks great, very detailed with quite some examples. I have some more minor comments and request, the main one to not cause raises during the execution of the notebook such that one can execute the 'run all' command and go through all of it.

File: Notebooks/LineOfSightEffects/line-of-sight_effects_tutorial.ipynb (hunk 1 above)
Comment (nataliehogg) [reply Notebooks/LineOfSightEffects/line-of-sight_effects_tutorial.ipynb:1]: <p>Done, thanks.</p>

Comment (nataliehogg) [reply Notebooks/LineOfSightEffects/line-of-sight_effects_tutorial.ipynb:1]: <p>Added, thanks.</p><p> </p>

Comment (nataliehogg) [reply Notebooks/LineOfSightEffects/line-of-sight_effects_tutorial.ipynb:1]: <p>I removed the code cells which generate the message and just left the text mentioning when the errors occur.</p>

Comment (nataliehogg) [reply Notebooks/LineOfSightEffects/line-of-sight_effects_tutorial.ipynb:1]: <p>As above.</p>

Comment (nataliehogg) [reply Notebooks/LineOfSightEffects/line-of-sight_effects_tutorial.ipynb:1]: <p>LaTeX fixed and ref added, thanks.</p><p> </p>

Comment (nataliehogg) [reply Notebooks/LineOfSightEffects/line-of-sight_effects_tutorial.ipynb:1]: <p>Added, thanks.</p><p> </p>

Comment (nataliehogg) [reply Notebooks/LineOfSightEffects/line-of-sight_effects_tutorial.ipynb:1]: <p>Done, thanks.</p>

Comment (nataliehogg) [reply Notebooks/LineOfSightEffects/line-of-sight_effects_tutorial.ipynb:1]: <p>Done, thanks.</p>

**Comment by nataliehogg** [comment]
Thanks for the comments @sibirrer, I think I have addressed them all (after a short fight with the LaTeX rendering!)

**Comment by ajshajib** [comment]
This is a great tutorial notebook going over the new features in details, thanks @nataliehogg!

**Review by ajshajib — APPROVED** [review]

**Review by sibirrer — APPROVED** [review]
Thank you @nataliehogg !

**Pull request merged**